src/components/filesystem/index.js

```diff
@@ -400,8 +400,7 @@ module.exports = {
                             that.showSpinner = false;
 			});
                     }).exceptionally(function(throwable) {
-			progress.show = false;
-			that.errorTitle = 'Error moving file: ' + name;
+			that.errorTitle = 'Error moving file';
```

Comment:
what was the reason for removing the name from the error?

Reply:
because name is not defined and the errorBody (line below) contains the filename